abs.yaml

```diff
@@ -0,0 +1,4 @@
+# the conda-build parameters to use for disabling --skip-existing
+build_parameters:
+  - "--suppress-variables"
```

Comment:
Why is the over-linking check disabled?
The recipe as it is depends heavily on run_exports being set in packages listed in the host section. I think this check still has its place.

Reply:
fair point.  I am used to disable normally both of them.  I re-added the overlinking check